### PR TITLE
Improve battle start component 

### PR DESCRIPTION
**Pull request by DjamelSoualmi**
[Ticket](https://trello.com/c/YuLobaQ6)

This Pr improves start battle component design 

Old one 

image

New One : 
Web 

battle-web-finalm

Tablet 

battle-tablet-3

mobile

image

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coorpacademy/components",
-  "version": "9.33.1",
+  "version": "9.32.13",
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/package.json:3]: il y a un souci.` rebase master`

File: packages/@coorpacademy-components/src/molecule/dashboard/start-battle/index.js [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
             </div>
-            <div
-              className={style.label}
-              // eslint-disable-next-line react/no-danger
-              dangerouslySetInnerHTML={{__html: label}}
-            />
-          </Link>
+            <div className={style.challengeText}>{challengeText}</div>
+            <Link className={style.button} onClick={onClick} href={href}>
+              <div>{startBattleText}</div>
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/dashboard/start-battle/index.js:32]:
```suggestion
              {startBattleText}
```

File: packages/@coorpacademy-components/src/molecule/dashboard/start-battle/style.css [hunk 3]
```diff
@@ -10,149 +10,298 @@
...
 .container {
   width: 100%;
+  max-width: 1080px;
+  height: 214px;
   display: flex;
   justify-content: center;
-  padding: 10px 0;
+  position: relative;
+  z-index: 0;
+  background: #111026;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/dashboard/start-battle/style.css:19]: rajoute dans colors.js `battle_bg`

File: packages/@coorpacademy-components/src/molecule/dashboard/start-battle/style.css [hunk 4]
```diff
@@ -10,149 +10,298 @@
...
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+  text-align: center;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #111026;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/dashboard/start-battle/style.css:155]: idem
rajoute dans colors.js battle_bg

File: packages/@coorpacademy-components/src/molecule/dashboard/start-battle/style.css [hunk 5]
```diff
@@ -10,149 +10,298 @@
...
 
-@media mobile {
-  .button {
-    margin-top: 20px;
+@media tablet {
+
+  .smallFlashRight,
+  .smallFlashLeft {
+      visibility: hidden;
+  width: 0;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/dashboard/start-battle/style.css:233]: indentation

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by esa-coorp — APPROVED** [review]

**Review by esa-coorp — COMMENTED** [review]
@DjamelSoualmi vérifie sur ie11

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**